tests/python/test_copy_result.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Test copying solver result object
+import aligator
+import aligator.dynamics
+import aligator.manifolds
+import numpy as np
+import example_robot_data as erd
+
+import copy
+
+robot = erd.load("ur5")
+rmodel = robot.model
+rdata = robot.data
+space = aligator.manifolds.MultibodyPhaseSpace(rmodel)
```

Comment:
This test seems to require Pinocchio _and_ `example-robot-data`, which isn't right since it concerns a general feature. I'll change this to Euclidean space, and the dynamics accordingly.